docs/tutorial/spark-tutorial/t-spark-streaming.md

```diff
@@ -0,0 +1,138 @@
+## Streaming workloads with Charmed Spark
+
+Spark comes with a built-in support for streaming workloads and Charmed Spark takes it even a step further by making it easy to integrate with streaming platform like Kafka.
+
+In this tutorial, we are going to use `juju` to deploy a Kafka setup, where a simple Juju charm will generate events to Kafka, and a Spark job we submit will consume these in real time to process it. We have already installed `juju` in previous sections.
+
+First of all, let's start by creating a fresh `juju` model to be used as a workspace for this streaming tutorial.
+
+```bash
+juju add-model spark-streaming
+```
+
+Now, let's deploy Zookeeper and Kafka using the `zookeeper-k8s` and `kafka-k8s` charms. Single units of these charm should be enough.
```

Comment:
For all the charms that we have here, it is probably a good idea to add the charmhub links, such that the user can go and find out more, if they are interested.

Reply:
Noted.